### DISCUSSION TITLE
How do I see what props a component has?

### DISCUSSION THREAD

**Discussion by alvarlagerlof** [ariakit/ariakit#1908] [Questions] [answered]
I'm exploring the api trying to figure out what props are available. Autocompletion works, but cmd+clicking on the component brings me to something like:

```tsx
export declare type Component<O extends Options> = {
    <T extends As>(props: Omit<O, "as"> & Omit<HTMLProps<Options<T>>, keyof O> & Required<Options<T>>): JSX.Element | null;
    (props: Props<O>): JSX.Element | null;
    displayName?: string;
};
```

Can I've managed to dig a bit deeper, but sadly I quickly get lost. Is there api docs or some kind of list somewhere?

**Comment by diegohaz** [comment, marked as answer]
There are no docs yet. But you can see the props in the `ts` folder: https://unpkg.com/browse/ariakit@2.0.0-next.41/ts/
